features/step_definitions/stats/provider_charts.rb

```diff
@@ -32,6 +32,26 @@
   end
 end
 
+Then(/^testing the analytics chart$/) do
+  page.document.synchronize(Capybara.default_max_wait_time,
+                            errors: [Cucumber::Ast::Table::Different, Selenium::WebDriver::Error::JavascriptError]) do
+    values = page.evaluate_script <<-JS
+      (function(){
+        var $chart = $('#mini-charts > .charts')
+        var series = $chart.data().chart.data()
```

Comment:
All I got so far is this (but to be honest, I have no idea of what I am doing 😓 
![image](https://user-images.githubusercontent.com/11318903/49169095-546e4b80-f339-11e8-9ffe-7785129637e3.png)